test/configuration_wire_protocol_test.dart

```diff
@@ -39,7 +39,7 @@ void main() {
       );
 
       // Verify the flags
-      expect(config.flags.length, equals(6));
+      expect(config.flags.length, equals(7));
```

Comment:
new test got added: https://github.com/Eppo-exp/sdk-test-data/commit/a59d0664b884918cbb213ef2265ec6e07f343057

Reply:
may want to just check that the length is greater than some reasonable amount, or that the loaded flag keys contain at least the key of the flags checked below.